packages/flutter_driver/lib/driver_extension.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// @dart = 2.11
```

Comment:
Why this?

Reply:
weird, I had some problems getting vs code to show analysis warnings locally - maybe this isn't necessary?

Reply:
AFAIK, it shouldn't be necessary since you raised the SDK constraints in the pubspec?